data/human/culture conversations.txt

```diff
@@ -524,3 +524,69 @@ mission "News Reporter - Paradise"
 			`	"We are fortunate that many people feel this way! Thank you for your answers, and for taking the time to do this interview!" she says.`
 			`	The drones dart back to follow her as she continues through the spaceport.`
 				decline
+
+mission "Scientists at the Pub after a Conference"
+	minor
+	source
+		government "Republic"
+		attributes "urban"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`As you take a break after a long journey and a longer-feeling landing procedure, you notice a group of people enter the pub where you're sitting, streaming in from the conference complex across the street. They vary in age, dress, and accent, but most have an identical badge dangling from a lanyard around their necks. You notice one badge that's been dropped on the floor, reading  "Attendee - 35th Annual Pan-galactic Conference on Interstellar Microbial Genomics and Population Dynamics". The group takes their seats at a long table near to you.`
+			choice
+				`	(Order a meal and listen in to their conversation.)`
+					goto "overheard scientists 1" 
+				`	(Find somewhere quieter to relax.)`
+					decline
+			label "overheard scientists 1"
+			`	Fragments of conversation make their way to you through the din of the pub.`
+			` `
+			`	Two of the younger attendees are looking a bit nervous.`
+			`	"I'm so glad to have my presentation over with. I was so worried about having to present on the first day, but now I can finally just relax and actually listen to the other talks!"`
```

Comment:
Moving the description and first lines of dialogue into a single line; line breaks are meant to be after changes in speaker.
```suggestion
			`	Two of the younger attendees are looking a bit nervous. "I'm so glad to have my presentation over with. I was so worried about having to present on the first day, but now I can finally just relax and actually listen to the other talks!"`
```